runtime/autoload/provider/clipboard.vim

```diff
@@ -67,7 +67,8 @@ function! provider#clipboard#Error() abort
 endfunction
 
 function! provider#clipboard#Executable() abort
-  if exists('g:clipboard')
+  " Setting g:clipboard to v:false explicitly opts-in to using the "builtin" clipboard providers below
+  if exists('g:clipboard') && g:clipboard != v:false
```

Comment:
If `g:clipboard` is defined then this gives an error because we can't compare a Dictionary to boolean.

cc @seandewar

Reply:
`:h type()`?